schemas/site.json

```diff
@@ -37,6 +37,44 @@
 				"type": "integer"
 			}
 		},
+		"archivedUsageStatsLogFiles": {
+			"type": "integer",
+			"description": "Defines what to do with the successfully processed and archived usage event log files. `0` will leave the files as they are. `1` will compress the files with gzip.",
+			"default": 0,
+			"validation": [
+				"in:0,1"
+			]
+		},
+		"enableGeoUsageStats": {
```

Comment:
Ah I see. Yeah I think I would prefer to make the values easier to understand, as long as it's just a little bit of extra code to validate the options at the context level.

Reply:
Can I use 'off' when geo locacion should not be enabled? -- I have problems with schema validation of the empty string...

Reply:
When I do it like this, and the admin chooses the first option (to disable the geo data), then the first radio button is not selected -- first when the page is reloaded it gets selected because of the default value. Also the setting is removed from the DB table.
Thus, I would prefer the value to be 'disabled' (or similar).